src/core/time.ts

```diff
@@ -134,6 +134,19 @@ export class TimeValue {
         return TimeValue.secsAndNs(s, ns);
     }
 
+    multiply(factor: number): TimeValue {
```

Comment:
Add a unit test for this.

Reply:
Added it here: https://github.com/lf-lang/reactor-ts/pull/106/commits/1aebda4ccb93d4add5f5936492ed44ea3faacd6d